### DISCUSSION TITLE
":Copilot setup" fails in VIM 9.0.1616

### DISCUSSION THREAD

**Discussion by MontyTHall** [community/community#57656] [Copilot Conversations]
### Select Topic Area

Question

### Body

VIM Version (not neovim): 9.0.1616
Node JS: v18.16.0

":Copilot setup" fails.  
Error detected while processing function copilot#Command[38]..function copilot#Command[30]..7[13]..copilot#Call[2]..<SNR>128_AgentCall[5]..<SNR>12
8_RequestAwait:
line    5:
E605: Exception not caught: copilot#agent(-32603): Request signInInitiate failed with message: connect ECONNREFUSED 140.82.114.3:443

I use a VPN and a proxy - if that's add'tl information.

Thanks

**Comment by tpope** [comment]
Hi @MontyTHall, see [the help for `g:copilot_proxy`](https://github.com/github/copilot.vim/blob/v1.9.1/doc/copilot.txt#L77-L91).

**Comment by We-unite** [comment]
Hello, I've met the same question with vim9.0.1927, but I don't ubderstand what the `g:copilot_proxy` really does.

My vim version: 9.0.1927
Nodejs version: v18.18.0

Error log:
```
处理 function copilot#Command[38]..function copilot#Command[30]..388[13]..copilot#Call[2]..<SNR>78_AgentCall[5]..<SNR>78_Requ
estAwait 时发生错误:
第    5 行:
E605: 异常没有被捕获: copilot#agent(-32603): Request signInInitiate failed with message: tunneling socket could not be establ
ished, cause=getaddrinfo EAI_AGAIN https
```

My vpn is Clash for windows, port is 7890.I've tried to run`:let g:copilot_proxy_strict_ssl = v:false` followed by `:let g:copilot_proxy = "127.0.0.1:7890"`.It doesn't work.

Waiting for solution.Thanks very much.
